README.rst

```diff
@@ -37,17 +37,16 @@ Pillow is the Friendly PIL Fork by `Alex Clark and Contributors <https://github.
 More Information
 ----------------
 
-- `Changelog <https://github.com/python-pillow/Pillow/blob/master/CHANGES.rst>`_
+- `Documentation <http://pillow.readthedocs.org/>`_
 
-  - `Pre-fork <https://github.com/python-pillow/Pillow/blob/master/CHANGES.rst#02b5---117-1995-2010>`_
+  - `Installation <http://pillow.readthedocs.org/installation.html>`_
+  - `Guides <http://pillow.readthedocs.org/guides.html>`_
```

Comment:
Minor thing: pillow.readthedocs.org is https, so these three can also be `https://pillow.readthedocs.org/...`